tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/fix_unmapped_survey_answers_test.py

```diff
@@ -0,0 +1,107 @@
+"""Original Issues: DC-1043, DC-1053 
+
+PPI answers are mapped to standard answer concepts in concept_relationship through 'Maps to value' (it has been this 
+case historically and it is still the case now), however, there are a bunch of PPI answer concepts (619 in total in 
+rdr20200807) missing such relationships in concept_relationship. Interestingly, the corresponding standard PPI 
+answer concepts could be found through 'Maps to'.  This might have been a bug in the vocabulary and they 
+probably should’ve used 'Maps to value' for mapping Answer concepts. 
+
+There are 619 unique answers (value_source_concept_ids) in observation that are mapped to a '0' value_as_concept_id, 
+out of which 368 are standard concepts, 202 are non-standard concepts that could be mapped to a standard concept 
+through Maps to in concept_relationship, and 49 are deprecated concepts that do not map to anything.
+
+For the 368 standard answers, we could just use it as-is for populating value_as_concept_id. However, among the 202 
+non-standard concepts, not all of the mapped standard concepts are classified as 'Answer' and they could belong to 
+other concept_classes. Below is a breakdown of the concept_class_id of the mapped concepts: 
+
+95 Context-dependent
+89 Answer
+8 Clinical Finding
+6 Question
+3 Unit
+1 Module
+
+Question or Module concept classes don't make sense so will get excluded. In conclusion, 
+
+1. For 368 standard concepts --> Set value_as_concept_id to the source_value_concept_id.
+2. For 49 deprecated concepts --> Set value_as_concept_id to 0. Actually we don't need to do anything for this case 
+because visit_as_concept_id is already 0.
+3, For 202 non-standard concepts --> Set value_as_concept_id to standard concept ids mapped through 'Maps to' 
+for the concept classes ('Answer', 'Context-dependent', 'Clinical Finding', 'Unit') only. 
```

Comment:
Do you feel strongly about repeating this comment? It will be easy for the two copies to get out of sync.

Reply:
I have no feeling for this comment, I will delete it.